android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -51,7 +51,10 @@ public void onCreate() {
 
     if (!Sentry.isEnabled()) {
       Log.d(TAG, "Initializing Sentry");
-      SentryAndroid.init(getApplicationContext());
+      SentryAndroid.init(getApplicationContext(), options -> {
+        options.setRelease("release-"+com.tavultesoft.kmea.BuildConfig.VERSION_NAME);
```

Comment:
Wouldn't this be "com.tavultesoft.kmapro" instead of "com.tavultesoft.kmea"?
Also for MainActivity.java